docs/quickstart/react.md

```diff
@@ -24,6 +24,8 @@ await app.listen(3000)
 With that, create a view at `views/index.jsx`:
 
 ```jsx
+import React from 'react'
```

Comment:
```suggestion
import * as React from 'react'
```
React team is planning to remove default import in future releases.
cf.: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports

Reply:
@darkgl0w thank you for spotting this! I'm updating the PR 🙂